src/commands/tenmans.ts

```diff
@@ -427,6 +432,18 @@ export async function handleButton(interaction: ButtonInteraction, db: Db) {
   command.execute();
 }
 
+async function fetchQueueUsers() {
+  // Fetch user data in batch so that mobile clients will cache (?)
+  const queuedUserIds = tenmansQueue.map(memberData => memberData.discordId)
+
+  await this.interaction.guild.members.fetch({
```

Comment:
Mix of async handling methods here: either you should be using async/await with try/catch, or the Promise API w/ .then/.catch